### PR TITLE
Saving images from url with special characters #1921

### PR DESCRIPTION
**Pull request by Piioo**
#1921 Now it is possible to save images from url with special characters like http://www.foo/öäü.jpg

### CODE REVIEW

File: spec/paperclip/io_adapters/http_url_proxy_adapter_spec.rb [hunk 1]
```diff
@@ -98,4 +98,15 @@
     end
   end
 
+  context "a url with special characters in the filename" do
+    before do
```
Comment (jyurek) [inline spec/paperclip/io_adapters/http_url_proxy_adapter_spec.rb]: Can you put all of this into one test instead of a before block?

**Comment by jyurek** [comment]
One quick comment, and if you can handle the Hound complaints, I'll totally pull this in. Thanks!

**Comment by Piioo** [comment]
@jyurek I don't get this errors on my local machine.

**Comment by lenart** [comment]
Why is this not merged? Having the same problems with UTF-8 characters in the URI (e.g. `http://www.foo/öäü.jpg`)

**Comment by tute** [comment]
Thanks for your work! @Piioo, can you please squash your commits together and rebase on top of latest master? If CI is green, we'll merge. Thanks! ❤️

**Comment by tute** [comment]
I've got a passing CI build, will rebase myself: https://travis-ci.org/thoughtbot/paperclip/builds/128930480. Thank you!

**Pull request closed without merging**